<launch>
	<node pkg="blockChainPack_" type="listener.py" name="publishingBlockStream" output="screen">
	</node>
</launch>
